tests/projects/embed/gnu-rm/hello/.xmake/windows/x64/xmake.conf

```diff
@@ -0,0 +1,19 @@
+{
+    __toolchains_cross_arm = {
+        "envs",
+        "cross"
+    },
+    arch = "arm",
+    bin = [[C:\NXP\S32DS.3.5\S32DS\build_tools\gcc_v10.2\gcc-10.2-arm32-eabi]],
+    buildir = "build",
+    ccache = true,
+    cross = "arm-none-eabi-",
+    host = "windows",
+    kind = "binary",
+    menu = true,
+    mode = "debug",
+    ndk_stdcxx = false,
+    plat = "cross",
+    sdk = [[C:\NXP\S32DS.3.5\S32DS\build_tools\gcc_v10.2\gcc-10.2-arm32-eabi]],
+    target_os = "none"
```

Comment:
please remove it.

Reply:
I understand the xmake.conf should not be here if follow the previous rule. If this xmake.conf is removed, the new comers don't know how to start. Do you have any good idea to remind the new comers how to configure the environment for gnu-rm?

Reply:
xmake.conf is only an internal cache file and it can be changed at any time.

@see https://github.com/xmake-io/xmake/blob/master/tests/projects/package/toolchain_gnu_rm/xmake.lua

we just run `xmake`

Reply:
or pass `--sdk=`

```console
xmake f -p cross --toolchain=gnu-rm --sdk=/xxxx
xmake
```